Tweetbok.IntegrationTest/PostsControllerTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FluentAssertions;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using TweetBook.Contract;
+using TweetBook.Domain;
+using Xunit;
+
+namespace Tweetbok.IntegrationTest
```

Comment:
Please rename the namespace and folder to IntegrationTests since we usually write more than one test.
So you will rename the file and the class named IntegrationTest.
Also, add another test here since this one is a little lonely here.

Reply:
Maybe add a test for the error case also.

Reply:
And one for checking the response when it's not empty.